plugins/out_kafka/librdkafka-0.11.6/src/rdkafka_proto.h

```diff
@@ -450,7 +450,7 @@ int rd_kafkap_bytes_cmp_data (const rd_kafkap_bytes_t *a,
 typedef struct rd_kafka_buf_s rd_kafka_buf_t;
 
 
-#define RD_KAFKA_NODENAME_SIZE  128
+define RD_KAFKA_NODENAME_SIZE  256
```

Comment:
Can this line be built? It seems that it contains invalid....

Reply:
I don't see any issues with the syntax

Reply:
Seems you are write